docs/en/modules/enduser-setup.md

```diff
@@ -1,13 +1,14 @@
 # enduser setup Module
 | Since  | Origin / Contributor  | Maintainer  | Source  |
 | :----- | :-------------------- | :---------- | :------ |
-| 2015-09-02 | [Robert Foss](https://github.com/robertfoss) | [Robert Foss](https://github.com/robertfoss) | [enduser_setup.c](../../../app/modules/enduser_setup.c)|
+| 2015-09-02 | [Robert Foss](https://github.com/robertfoss), [Jason Follas](https://github.com/jfollas) | [Robert Foss](https://github.com/robertfoss) | [enduser_setup.c](../../../app/modules/enduser_setup.c)|
```

Comment:
I know you didn't! It's good that you added your name to the source code in an earlier contribution.
If you have good ideas as for how to give credit to those who contribute significantly I'm sure we'd welcome those as well. [GitHub's charts](https://github.com/nodemcu/nodemcu-firmware/graphs/contributors) are nice but just a machine's point of view of course.